cni/network/invoker.go

```diff
@@ -27,8 +27,16 @@ type IPAMAddConfig struct {
 }
 
 type IPAMAddResult struct {
-	ipv4Result       *cniTypesCurr.Result
-	ipv6Result       *cniTypesCurr.Result
+	defaultCniResult CNIResult
+	cniResults       []CNIResult
 	ncResponse       *cns.GetNetworkContainerResponse
 	hostSubnetPrefix net.IPNet
+	ipv6Enabled      bool
+}
+
+type CNIResult struct {
+	ipResult           *cniTypesCurr.Result
+	addressType        string
```

Comment:
Wouldn't the net.IP inside the IPConfig of the cni Result have the address type to be v4 or v6. Do we need another variable for it or is this being used for something else?